src/main/java/bhs/devilbotz/subsystems/Arm.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package bhs.devilbotz.subsystems;
+
+import bhs.devilbotz.Constants;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Arm extends SubsystemBase {
+  /** Creates a new Arm. */
+  private WPI_TalonSRX armMotor;
+
+  private boolean armUp;
+
+  public Arm() {
+    armMotor = new WPI_TalonSRX(Constants.ARM_CAN_ID);
+    armUp = false;
+  }
+
+  public boolean isArmUp() {
+    return armUp;
+  }
```

Comment:
How would "isArmUp()" will be used.  Can we remove it until we absolutely need it?

Reply:
I agree, it also has faults like I said in my previous comment:
> this boolean will be true even if the motor is not up, it could be down and the driver could click the button by accident and it still be down but true. Maybe use an encoder instead?